asteroid/models/dcunet.py

```diff
@@ -42,10 +42,12 @@ def __init__(
         masker = self.masknet_class.default_architecture(architecture, **(masknet_kwargs or {}))
         super().__init__(encoder, masker, decoder)
 
-    def postprocess_encoded(self, tf_rep):
+    def forward_encoder(self, wav):
+        tf_rep = self.encoder(wav)
```

Comment:
Yes, or `return complex_nn.as_torch_complex(super().forward_encoder(wav))` and below `from_torchaudio(torch.view_as_real(super().apply_masks(tf_rep, est_masks))`

I have no preference

Reply:
I prefer the current way, it's clearer.